common/static/common/js/components/views/list.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Generic view to render a collection.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'underscore'], function(Backbone, _) {
+        var ListView = Backbone.View.extend({
+            /**
+             * Override with the view used to render models in the collection.
+             */
+            itemViewClass: Backbone.View,
+
+            initialize: function (options) {
+                this.itemViewClass = options.itemViewClass || this.itemViewClass;
+                this.collection.on('add', this.render, this);
+                this.collection.on('remove', this.render, this);
+                this.collection.on('reset', this.render, this);
+                this.collection.on('sync', this.render, this);
+                this.collection.on('sort', this.render, this);
```

Comment:
I don't know if ``sort`` will work in combination with the paginator, since locally sorting the items won't give the correct result. Is the idea that this view might be used with a non-paginated collection?